
introduce -Wundef check to catch macro typos
more infos about motivation at https://sourceware.org/glibc/wiki/Wundef

patch 04/04: use -Wundef compiler option for future compile checks

Signed-off-by: Andreas Mohr <and@gmx.li>

--- a/m4.include/mc-cflags.m4
+++ b/m4.include/mc-cflags.m4
@@ -72,6 +72,7 @@
     MC_CHECK_ONE_CFLAG([-Wstrict-prototypes])
     MC_CHECK_ONE_CFLAG([-Wswitch])
     MC_CHECK_ONE_CFLAG([-Wtype-limits])
+    MC_CHECK_ONE_CFLAG([-Wundef])
     MC_CHECK_ONE_CFLAG([-Wuninitialized])
     MC_CHECK_ONE_CFLAG([-Wunused-but-set-variable])
     MC_CHECK_ONE_CFLAG([-Wunused-function])
